src/pages/contact.tsx

```diff
@@ -25,20 +96,63 @@ const Contact: React.FC = () => {
       <PageBody>
         <WholePageBox>
           <div className={styles.contactBody}>
-            <div className="contactUsSection">
-              <Typography variant="h4">Contact Us</Typography>
-              {/* TODO Update Link? */}
-              <Typography variant="body1">
-                You can contact us using{" "}
-                <a
-                  href="https://docs.google.com/forms/d/e/1FAIpQLSe1lAW158F3s0NFJEHn2Fsfo4goau4QQTLhlbX7cJJiXGS7xw/viewform"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  this form
-                </a>
-                .
-              </Typography>
+            <div className={styles.contactForm}>
+              <form onSubmit={handleSubmit}>
+                {alertInfo}
+                <div className={styles.contactName}>
+                  <div style={{ width: "100%" }}>
+                    <Typography variant="h6">First Name:</Typography>
```

Comment:
Make this component a label using the 'component' attribute that comes with typography. See [mui documentation](https://mui.com/material-ui/react-typography/#changing-the-semantic-element). Give each input an id and use the 'htmlFor' attribute on the label to make the form more accessible.